### PR TITLE
Renamed connectiontracker from app-> originator network->destination

### PR DESCRIPTION
**Pull request by amitlimaye**

### CODE REVIEW

File: enforcer/datapath.go [hunk 1]
```diff
@@ -36,13 +36,13 @@ type datapathEnforcer struct {
 	contextTracker cache.DataStore
 	puTracker      cache.DataStore
 	// Key=FlowHash Value=Connection. Created on syn packet from network with regular flow hash
-	networkConnectionTracker cache.DataStore
+	destinationConnectionTracker cache.DataStore
```
Comment (satyamsi) [inline enforcer/datapath.go]: destination should be net

File: enforcer/datapath.go [hunk 2]
```diff
@@ -36,13 +36,13 @@ type datapathEnforcer struct {
...
 	// Key=FlowHash Value=Connection. Created on syn packet from application with regular flow hash
-	appConnectionTracker cache.DataStore
+	originatorConnectionTracker cache.DataStore
 	// Key=Context Value=Connection. Create on syn packet from application with local context-id
 	contextConnectionTracker cache.DataStore
 
-	sourcePortCache      cache.DataStore
+	originatorPortCache  cache.DataStore
```
Comment (satyamsi) [inline enforcer/datapath.go]: originatorSourcePortCache

File: enforcer/datapath.go [hunk 3]
```diff
@@ -36,13 +36,13 @@ type datapathEnforcer struct {
 	contextTracker cache.DataStore
 	puTracker      cache.DataStore
 	// Key=FlowHash Value=Connection. Created on syn packet from network with regular flow hash
-	networkConnectionTracker cache.DataStore
+	destinationConnectionTracker cache.DataStore
 	// Key=FlowHash Value=Connection. Created on syn packet from application with regular flow hash
-	appConnectionTracker cache.DataStore
+	originatorConnectionTracker cache.DataStore
 	// Key=Context Value=Connection. Create on syn packet from application with local context-id
 	contextConnectionTracker cache.DataStore
```
Comment (satyamsi) [inline enforcer/datapath.go]: prefix with originator

File: enforcer/datapath.go (hunk 1 above)
Comment (satyamsi) [inline enforcer/datapath.go]: netConnectionTracker

File: enforcer/datapath.go [hunk 4]
```diff
@@ -36,13 +36,13 @@ type datapathEnforcer struct {
 	contextTracker cache.DataStore
 	puTracker      cache.DataStore
 	// Key=FlowHash Value=Connection. Created on syn packet from network with regular flow hash
-	networkConnectionTracker cache.DataStore
+	destinationConnectionTracker cache.DataStore
 	// Key=FlowHash Value=Connection. Created on syn packet from application with regular flow hash
-	appConnectionTracker cache.DataStore
+	originatorConnectionTracker cache.DataStore
```
Comment (satyamsi) [inline enforcer/datapath.go]: appConnectionTracker

File: enforcer/datapath.go (hunk 3 above)
Comment (satyamsi) [inline enforcer/datapath.go]: appContextConnectionTracker

File: enforcer/datapath.go [hunk 5]
```diff
@@ -36,13 +36,13 @@ type datapathEnforcer struct {
...
 	// Key=Context Value=Connection. Create on syn packet from application with local context-id
 	contextConnectionTracker cache.DataStore
 
-	sourcePortCache      cache.DataStore
+	originatorPortCache  cache.DataStore
 	destinationPortCache cache.DataStore
```
Comment (satyamsi) [inline enforcer/datapath.go]: netDestinationPortCache

**Comment by dstiliadis** [comment]
This is stale and we have changed the naming anyway. I am closing.

**Pull request closed without merging**